### PR TITLE
Support Debian9 and CentOS7 in Github Actions

### PR DESCRIPTION
**Pull request by ShigekiKarita**
As circle CI supports those OSs and stopped due to short of 2500 free credits
[image](
https://app.circleci.com/settings/plan/github/espnet/overview?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fespnet)
I propose Github Actions based redundant CI by its similar functionality https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer

### CODE REVIEW

File: egs/mini_an4/asr1/cmd.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 
 # Select the backend used by run.sh from "local", "stdout", "sge", "slurm", or "ssh"
-cmd_backend='local'
+cmd_backend='stdout'
```
Comment (kan-bayashi) [inline egs/mini_an4/asr1/cmd.sh:31]: I remembered `stdout` does not stop when the error happened.
Is it OK?

Comment (ShigekiKarita) [reply egs/mini_an4/asr1/cmd.sh:31]: I found it was not true. Now stdout.pl correctly stopped when the exist status != 0. See  https://github.com/espnet/espnet/runs/1105039198#step:9:46

I suspect that maybe we forgot `set -euo pipefail`

Comment (ShigekiKarita) [reply egs/mini_an4/asr1/cmd.sh:31]: Well, however, I'm not sure. I will revert this.

**Review by kan-bayashi — APPROVED** [review]
LGTM

**Pull request merged**